spanner_orm/tests/decorator_test.py

```diff
@@ -0,0 +1,96 @@
+from unittest import mock
+# python3
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import parameterized
+from spanner_orm import decorator
+
+
+# pylint: disable=redundant-keyword-arg
+# pylint: disable=no-value-for-parameter
```

Comment:
what is causing these lint warnings? can we update the tests so we don't have to disable these, or at least cut down the amount of code for which these warnings are disabled?